cmd/mtaBuild_test.go

```diff
@@ -43,7 +46,13 @@ func TestMarBuild(t *testing.T) {
 		existingFiles["package.json"] = "{\"name\": \"myName\", \"version\": \"1.2.3\"}"
 		fileUtils := MtaTestFileUtilsMock{existingFiles: existingFiles}
 
-		err := runMtaBuild(options, &cpe, &e, &fileUtils, &httpClient)
+		utils := newNpmMockUtilsBundle()
```

Comment:
I doubt we should use this method from `npmExecuteScripts_test`

Reply:
It was just a means to not duplicate the code, since `newNpmMockUtilsBundle()` does not do anything customized for the npmExecuteScripts step. It simply instantiates the FilesMock and mockExecRunner for testing.